CMSgov/cms-carts-seds
# mdct-2320

### Description
Headers in charts in Section 5 for 2022 reports were mislabeled and causing errors. Updated the headers to correct years.


### Related ticket(s)


MDCT-2320

---
### How to test
Spin up CARTS, log in as stateuser2 and enter into the 2022 report. Go to Section 5 and see that the headers in the charts are '2022, 2023, 2024' and the numbers that entered show up correctly in the charts.


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
